src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3262,4 +3262,22 @@
   <data name="SecurityUnableToDisplayPermissions" xml:space="preserve">
     <value>Unable to display permissions.</value>
   </data>
+  <data name="StashChanges" xml:space="preserve">
+    <value>Stash changes</value>
+  </data>
+  <data name="DiscardChanges" xml:space="preserve">
+    <value>Discard changes</value>
+  </data>
+  <data name="BringChanges" xml:space="preserve">
+    <value>Bring changes</value>
```

Comment:
```suggestion
    <value>Bring my changes to</value>
```
We should display the branch name after the word "to". Let me know if you want me to work on this change.

Reply:
It would be great if you could help me. I have not much time these days...